### ISSUE TITLE
use nni on colab:{"error":"File not found: /content/log/mo5q17fb/trials/L9du6/trial.log"}

### ISSUE THREAD

**Issue by ABChh26** [microsoft/nni#5122]
**Describe the issue**:
When i was using nni on colab , it always shown tip: {"error":"File not found: /content/log/mo5q17fb/trials/L9du6/trial.log"}

**Environment**:
- NNI version:
- Training service (local|remote|pai|aml|etc):
- Client OS:
- Server OS (for remote mode only):
- Python version:
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?:
- Is running in Docker?:

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by ABChh26** [comment]
{"error":"File not found: /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log"}

**Comment by ultmaster** [comment]
Have you tried to `cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log`. Does it actually exist or not?

**Comment by Lijiaoa** [comment]
hi @ABChh26 Could you help confirm the log directory?

**Comment by ABChh26** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        设置成你想放置的文件夹即可，format：    experimentWorkingDirectory: "xx"
xx最好是绝对路径
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                2351428409
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 9/22/2022 ***@***.***> wrote： 



hi @ABChh26 Could you help confirm the log directory?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by Lijiaoa** [comment]
why did you write a few lines of css code? I think it's unuseful. You should reply this question. @ABChh26 

> Have you tried to `cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log`. Does it actually exist or not?

**Comment by Lijiaoa** [comment]
> 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径

其实我想跟你确认的是这个事情，当你执行命令 `cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log` 给你反馈的结果是什么呢？

**Comment by siwuxei** [comment]
> 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径

这个方法我尝试过，没有解决问题，在终端执行py文件，可以看到nni返回了中间结果，但就是生成不了trial.log文件，更换nni版本也不行，以前我使用nni==2.5.0是可以的，现在不行了

**Comment by ABChh26** [comment]
>

**Comment by ABChh26** [comment]
> > 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径
> 
> 其实我想跟你确认的是这个事情，当你执行命令 `cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log` 给你反馈的结果是什么呢？

没尝试过

**Comment by ABChh26** [comment]
> 



> > 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径
> 
> 这个方法我尝试过，没有解决问题，在终端执行py文件，可以看到nni返回了中间结果，但就是生成不了trial.log文件，更换nni版本也不行，以前我使用nni==2.5.0是可以的，现在不行了

可能还是路径问题吧，默认情况下，nni-experiments这个文件夹不在content下，我这个问题可能也是误打误撞解决了，抱歉啊家人！这个是我的config.yml，我每次都是路径切换到train_reg.py所在文件夹，然后在ipynb文件中运行：
searchSpaceFile: space.json
experimentName: pyg
experimentWorkingDirectory: "/content/drive/MyDrive/exp"
trialCommand: python train_reg.py 
trialGpuNumber: 1
trialConcurrency: 2
maxExperimentDuration: 24h
tuner:
  name: TPE
  classArgs:
    optimize_mode: maximize


trainingService:
  platform: local
  useActiveGpu: true
  maxTrialNumberPerGpu: 2

**Comment by ABChh26** [comment]
> > > 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径
> > 
> > 
> > 其实我想跟你确认的是这个事情，当你执行命令 `cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log` 给你反馈的结果是什么呢？
> 
> 没尝试过

会员过期了……

**Comment by siwuxei** [comment]
> > 
> 
> > > 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径
> > 
> > 
> > 这个方法我尝试过，没有解决问题，在终端执行py文件，可以看到nni返回了中间结果，但就是生成不了trial.log文件，更换nni版本也不行，以前我使用nni==2.5.0是可以的，现在不行了
> 
> 可能还是路径问题吧，默认情况下，nni-experiments这个文件夹不在content下，我这个问题可能也是误打误撞解决了，抱歉啊家人！这个是我的config.yml，我每次都是路径切换到train_reg.py所在文件夹，然后在ipynb文件中运行： searchSpaceFile: space.json experimentName: pyg experimentWorkingDirectory: "/content/drive/MyDrive/exp" trialCommand: python train_reg.py trialGpuNumber: 1 trialConcurrency: 2 maxExperimentDuration: 24h tuner: name: TPE classArgs: optimize_mode: maximize
> 
> trainingService: platform: local useActiveGpu: true maxTrialNumberPerGpu: 2

这个问题我已经解决了， 请查看#5122，问题在于{trialGpuNumber: 1，trialConcurrency: 2}设置，请调整该配 {trialGpuNumber: 1， trialConcurrency: 1}即可，该问题可关闭

**Comment by Lijiaoa** [comment]
> > > 
> > 
> > 
> > > > 设置成你想放置的文件夹即可，format： experimentWorkingDirectory: "xx" xx最好是绝对路径
> > > 
> > > 
> > > 这个方法我尝试过，没有解决问题，在终端执行py文件，可以看到nni返回了中间结果，但就是生成不了trial.log文件，更换nni版本也不行，以前我使用nni==2.5.0是可以的，现在不行了
> > 
> > 
> > 可能还是路径问题吧，默认情况下，nni-experiments这个文件夹不在content下，我这个问题可能也是误打误撞解决了，抱歉啊家人！这个是我的config.yml，我每次都是路径切换到train_reg.py所在文件夹，然后在ipynb文件中运行： searchSpaceFile: space.json experimentName: pyg experimentWorkingDirectory: "/content/drive/MyDrive/exp" trialCommand: python train_reg.py trialGpuNumber: 1 trialConcurrency: 2 maxExperimentDuration: 24h tuner: name: TPE classArgs: optimize_mode: maximize
> > trainingService: platform: local useActiveGpu: true maxTrialNumberPerGpu: 2
> 
> 这个问题我已经解决了， 请查看#5146，问题在于{trialGpuNumber: 1，trialConcurrency: 2}设置，请调整该配 {trialGpuNumber: 1， trialConcurrency: 1}即可，该问题可关闭

@ABChh26 you could follow this suggestion to fix your issue. Thanks!

**Comment by ABChh26** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            没有，但是我在config.yml文件里设置了实验工作路径(experimentWorkingDirectory)解决了这个问题
        
    
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                2351428409
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 9/13/2022 10:43，Yuge ***@***.***> wrote： 



Have you tried to cat /root/nni-experiments/rgiu2kbx/trials/rQUUZ/trial.log. Does it actually exist or not?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by Lijiaoa** [comment]
I'll close this issue because this problem had been resolved. If you still have any questions please feel free to reopen it. Thanks.
